src/index.html

```diff
@@ -1,15 +1,203 @@
-<body>
-  <h1>HTML Form</h1>
-  <script type="text/javascript" src="./main.js"></script>
-</body>
+
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>HTML Form</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    autocomplete="off"
+    >
+      <fieldset>
+        <legend>Personal information</legend>
+
+        <label class="form-field" for="surnmae">
+          Surname:
+          <input
+            type="text"
+            id="surnmae"
+            name="surnmae"
+            required
+          >
+        </label>
+
+        <label class="form-field" for="name">
+          Name:
+          <input
+            type="text"
+            id="name"
+            name="name"
+            required
+          >
+        </label>
+
+        <label class="form-field" for="age">
+          How old are You?
+          <input
+            type="number"
+            id="age"
+            name="age"
+            value="12"
+            min="1"
+            max="100"
+            required
+          >
+        </label>
+
+        <label class="form-field" for="birth">
+          Full date of birth:
+          <input
+            type="date"
+            id="birth"
+            name="birth"
+            required
+          >
+        </label>
+
+        <label class="form-field" for="term">
+          I accept the term of the agreement
+          <input
+            type="checkbox"
+            id="term"
+            name="term"
+            required
+          >
+        </label>
+
+      </fieldset>
+
+      <fieldset>
+        <legend>Registration</legend>
+
+        <label class="form-field" for="e-mail">
+          E-mail:
+          <input
+            type="email"
+            placeholder="email@example.com"
+            name="e-mail"
+            id="e-mail"
+            required
+          >
+        </label>
+
+        <label class="form-field" for="password">
+          Password:
+          <input
+            type="password"
+            name="password"
+            id="password"
+            required
+          >
+        </label>
+
+      </fieldset>
+
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form-field" for="cats">
+          Do you love cats?
+          <label for="love-cats">
+            <input
+              type="radio"
+              id="love-cats"
+              name="yes"
+              value="yes"
+            >Yes
+          </label>
+
+          <label for="hate-cats">
+            <input
+              type="radio"
+              id="hate-cats"
+              name="no"
+              value="no"
+            >No
+          </label>
```

Comment:
name for a group of radio buttons should be the same, rename to 'love-cats', cause now it's possible to choose 2 options at a time
```suggestion
        <div class="form-field" for="cats">
          Do you love cats?
          <label for="love-cats">
            <input
              type="radio"
              id="love-cats"
              name="love-cats"
              value="yes"
            >Yes
          </label>

          <label for="hate-cats">
            <input
              type="radio"
              id="hate-cats"
              name="love-cats"
              value="no"
            >No
          </label>
```